### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Very sorry but I can't build it on my linux because my packages are too old. I can check it this evening though.

### CODE REVIEW

**Comment by sdroege** [comment]
It's failing because the code got some new "extern crate" but the Cargo.toml was not updated. However those crates are actually not used, so shouldn't be there to begin with.

Also this update is "breaking" the versions in build.rs and Cargo.toml. It is removing some version features, which are there for consistency (everything has features for the major versions, 1.0, 1.2, 1.4, etc, independent of whether API was added or not... to make it simpler for users to select their required version without having to think/check).

**Comment by GuillaumeGomez** [comment]
I'll clean this PR this evening then. :)

**Comment by GuillaumeGomez** [comment]
Updated, sorry for the delay. There is still an issue remaining for the debug generation in gir (for cases like `[c_int; 64]`, no `Debug`...).

**Comment by GuillaumeGomez** [comment]
This issue should be fixed by https://github.com/rust-lang/rust/pull/46445.

File: gstreamer-app-sys/src/lib.rs [hunk 1]
```diff
@@ -1,10 +1,12 @@
-// This file was generated by gir (94e079d) from gir-files (???)
+// This file was generated by gir (0113735) from gir-files (???)
 // DO NOT EDIT
 
 #![allow(non_camel_case_types, non_upper_case_globals)]
 
 extern crate libc;
+#[macro_use] extern crate bitflags;
 extern crate glib_sys as glib;
+extern crate gobject_sys as gobject;
```
Comment (sdroege) [inline gstreamer-app-sys/src/lib.rs]: This will cause warnings about unused crates

File: gstreamer-app-sys/Cargo.toml [hunk 2]
```diff
@@ -2,8 +2,10 @@
 pkg-config = "0.3.7"
 
 [dependencies]
+bitflags = "1.0"
 libc = "0.2"
 glib-sys = { git = "https://github.com/gtk-rs/sys" }
+gobject-sys = { git = "https://github.com/gtk-rs/sys" }
```
Comment (sdroege) [inline gstreamer-app-sys/Cargo.toml]: Unused crates

**Comment by GuillaumeGomez** [comment]
@sdroege: Fixed and updated.

**Comment by sdroege** [comment]
See the warnings in the travis build. The ones I listed were only some of the many :)
https://travis-ci.org/sdroege/gstreamer-sys/jobs/310871268#L533
https://travis-ci.org/sdroege/gstreamer-sys/jobs/310871268#L541
https://travis-ci.org/sdroege/gstreamer-sys/jobs/310871268#L643

File: gstreamer-audio-sys/Cargo.toml [hunk 3]
```diff
@@ -13,6 +13,9 @@ path = "../gstreamer-sys"
 [dependencies.gstreamer-base-sys]
 path = "../gstreamer-base-sys"
 
+[dependencies.gstreamer-tag-sys]
+path = "../gstreamer-tag-sys"
```
Comment (sdroege) [inline gstreamer-audio-sys/Cargo.toml]: Unused

File: gstreamer-pbutils-sys/Cargo.toml [hunk 4]
```diff
@@ -16,6 +16,9 @@ path = "../gstreamer-audio-sys"
 [dependencies.gstreamer-video-sys]
 path = "../gstreamer-video-sys"
 
+[dependencies.gstreamer-tag-sys]
+path = "../gstreamer-tag-sys"
```
Comment (sdroege) [inline gstreamer-pbutils-sys/Cargo.toml]: Unused

**Comment by GuillaumeGomez** [comment]
@sdroege: How come you see unused crates? I don't have any warnings or anything. :-/

**Comment by sdroege** [comment]
I don't know why there's no warning, but nothing of those crates is used anywhere :)

**Comment by GuillaumeGomez** [comment]
That's so strange. (I updated btw.)

**Comment by GuillaumeGomez** [comment]
🎊

**Pull request merged**